example/advection/advection_package.cpp

```diff
@@ -106,15 +107,16 @@ void PreFill(Container<Real> &rc) {
   int ie = pmb->ncells1 - 1;
   int je = pmb->ncells2 - 1;
   int ke = pmb->ncells3 - 1;
-  CellVariable<Real> &qin = rc.Get("advected");
-  CellVariable<Real> &qout = rc.Get("one_minus_advected");
-  for (int i = is; i <= ie; i++) {
-    for (int j = js; j <= je; j++) {
-      for (int k = ks; k <= ke; k++) {
-        qout(k, j, i) = 1.0 - qin(k, j, i);
-      }
-    }
-  }
+  PackIndexMap imap;
+  std::vector<std::string> vars({"advected", "one_minus_advected"});
+  auto v = PackVariables<>(rc, vars, imap);
+  const int in = imap["advected"].first;
+  const int out = imap["one_minus_advected"].first;
+  par_for(
+      "advection_package::PreFill", DevExecSpace(), ks, ke, js, je, is, ie,
```

Comment:
If I can get [this](https://github.com/lanl/parthenon/pull/77) merged soon these function calls would benefit. It would be easy to insert indices in the wrong order.